app/routes/users/components/UserProfile.js

```diff
@@ -135,12 +136,21 @@ export default class UserProfile extends Component<Props, UpcomingEventsProps> {
 
   renderFields() {
     const { user } = this.props;
-    const fields = Object.keys(fieldTranslations).filter(field => user[field]);
+    let fields = Object.keys(fieldTranslations).filter(field => user[field]);
```

Comment:
Why change this to let? :thinking:

Reply:
Needed for the first commit and forgot to change it. Just waited for review to fix it :smiley: